build.sbt

```diff
@@ -1317,7 +1317,7 @@ lazy val listenerApi = (project in file("ui/listener-api"))
   .settings(
     name := "nussknacker-listener-api",
   )
-  .dependsOn(restmodel)
+  .dependsOn(restmodel, deploymentManagerApi)
```

Comment:
I'm not sure that is good idea.. Let's talk with @arkadius about: https://nussknacker.io/documentation/docs/next/developers_guide/Basics#modules-architecture-and-conventions

Reply:
I understand, but because OnDeployActionSuccess uses DeploymentComment I have to make DeploymentComment accessible in ui-listener, I wrote to arkadius already to discuss how best to solve it

Reply:
For now we've decided to put DeploymentComment class in ui-listener and leaving the associated object in ui